app/controllers/communities_controller.rb

```diff
@@ -52,8 +54,24 @@ def set_community
     @community = Community.find(params[:id])
   end
 
+  def find_all_communities
+    @communities = Community.all
```

Comment:
Don't use .all method

Reply:
Communities will be displayed with an "infinite scroll feature" further